<launch>
	
	<node pkg="h4r_pan_tilt_adapter" name="pan_tilt_cam0_adapter" type="pan_tilt_adapter_node" output="screen" required="true">
		<param name="servo_rate" value="500"/> 
	</node>
	
	<node pkg="h4r_pan_tilt_adapter" name="pan_tilt_cam0_joy" type="pan_tilt_joy_node" output="screen" required="true">
		<param name="axis_pan" value="0" />
		<param name="axis_tilt" value="1" />
		
		<param name="button_tilt_up" value="19"/>
		<param name="button_tilt_down" value="21"/>
		<param name="button_pan_up" value="20"/>
		<param name="button_pan_down" value="22"/>
	    
		<remap from="~/joy" to="/joy"/>
	</node>
	
	<node pkg="joy" name="joystick" type="joy_node" />
	<node pkg="rviz" type="rviz" name="rviz" args=" -d $(find h4r_pan_tilt_adapter)/rviz/pan_tilt_view.rviz" required="true" />	
</launch>
